packages/synapse-react-client/src/components/FullWidthAlert/FullWidthAlert.tsx

```diff
@@ -116,50 +105,81 @@ function FullWidthAlert(props: FullWidthAlertProps) {
       }
     }
   }, [onClose, autoCloseAfterDelayInSeconds])
-  const additionalAlertVariantClass = variant ? `alert-${variant}` : ''
-  return (
+
+  const alert = (
     <Alert
-      variant={variant}
-      show={show}
-      dismissible={false}
-      transition={transition}
-      className={`FullWidthAlert bootstrap-4-backport ${
-        isGlobal ? 'global' : ''
-      } ${additionalAlertVariantClass}`}
+      severity={variantToSeverity(variant)}
+      sx={{
+        width: '100%',
+        my: '10px',
+        '.MuiAlert-message': {
+          flexGrow: 1,
+        },
+      }}
+      className="FullWidthAlert"
```

Comment:
Used for styling in displayToast and to adjust styling when SynapseNavDrawerisShowing